docs/source/get_involved/code_of_conduct.rst

```diff
@@ -342,14 +397,183 @@ Guide <https://www.djangoproject.com/conduct/>`__ and we are also
+
+
+A.2 Guidance on applying the CoC for resource allocation decisions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This section deals with resource allocation decisions by ``sktime`` community members,
+and criteria for decision making on resource allocation violating the code of conduct.
+
+Legal status
+^^^^^^^^^^^^
+
+This guidance of applying the CoC for resource allocation decisions impacts only
+adjudication by the CoC committee within the jurisdiction of the CoC committee.
+
+It does not impact the terms of the license by
+which ``sktime`` - the software package - is distributed.
+
+Out-Of-Scope
+^^^^^^^^^^^^
+
+Out of scope are spaces outside ``sktime`` jurisdiction as per Section 1.
+
+In-Scope
+^^^^^^^^
+
+In scope for this section are all resource allocation decisions that are made
+within ``sktime`` jurisdiction as per Section 1, including decisions that are:
+
+* made by elected members of the ``sktime`` community, i.e., core developers,
+  Community Council members, Code of Conduct committee members, or
+* made as part of an ``sktime`` activity or event.
+
+A resource allocation is in scope if it satisfies at least one of the following criteria:
+
+* any decisions that allocate resources primarily obtained due to the ``sktime`` project and the volunteer contributions of the community.
+  This includes research grants awarded directly for the benefit of the ``sktime`` project,
+  commercial opportunities arising from or directly directed to the ``sktime`` project,
+  and any activities that use the ``sktime`` brand as a primary branding.
+  This condition applies even if relevant decisions are made in personal roles,
+  or roles other than ``sktime`` official roles, as long as it falls within the general jurisdiction scope outlined above.
+* any decisions by members of the ``sktime`` developer community, in their execution of an official ``sktime`` role.
+  This especially includes core developer, Community Council, and Code of Conduct Committee roles.
+
+Examples for in-scope:
+
+* an academic allocating grant monies from a grant, or a data science consultant providing consultancy services, with a primary ``sktime`` branding
+* hiring decisions for roles that are advertised with a primary ``sktime`` branding
+* spending decisions from ``sktime`` community administered accounts
+* decisions arising from academic collaboration or commercial requests directed to the ``sktime`` community,
+  e.g., via official email, or via digital ``sktime`` discussion tools (slack, GitHub discussion etc)
+
+"resources" in the above context are to be interpreted in the usual sense of the word,
+i.e., valuable possessions, opportunities, qualities, that can be accessed and allocated.
+
+Examples include but are not limited to:
+
+* budget, money, financial credit
+* job opportunities, business opportunities
+* decision making power, soft or hard influence
+* dedicated work time of subordinates or dependents
```

Comment:
How are we going to control that, I don't understand. Is this implying that we're affecting how someone's distributing their subordinates time and if it's "supposed to" go on working on sktime we're somehow able to influence that? How is that sktime's business and not the employer of said individual whose subordinates are mentioned here?

Reply:
And re job opportunities - this is relevant imo only if the job opportunity was to sktime community. If someone messages a CC member with "Hi X, I see your contributions to sktime, I'd like to offer you a job" - it's not a "sktime resourse"